.gitignore

```diff
@@ -12,4 +12,3 @@ include/
 docs/_build
 docs/api
 docs/_templates
```

Comment:
You should add `docs/hsc_cosmos_35.sca` to the `.gitignore` file, since this is generated by the docs and we don't want to commit these changes. It might even be better to modify the docs to add it to a `docs/_temp` directory that will contain any dynamic files created by the docs that we can add to `.gitignore`.